src/visiomode/config.py

```diff
@@ -2,18 +2,17 @@
 
 #  This file is part of visiomode.
 #  Copyright (c) 2020 Constantinos Eleftheriou <Constantinos.Eleftheriou@ed.ac.uk>
+#  Copyright (c) 2024 Olivier Delree <odelree@ed.ac.uk>
```

Comment:
Yes nice please do remember to add this as you go, and we should also probably create an authors or contributors txt